.golangci.yml

```diff
@@ -95,6 +94,11 @@ issues:
 
   # Excluding configuration per-path, per-linter, per-text and per-source
   exclude-rules:
+    - path: _test\.go
+      linters:
+      - revive
+      - stylecheck
+      text: "should not use dot imports"
```

Comment:
Disabling this on test packages as it catches each `ginkgo` and `gomega` import.